packages/styles/scss/patterns/card-section/_card-section.scss

```diff
@@ -50,96 +36,61 @@
     &__title {
       @include type-style('expressive-heading-02');
 
+      padding-left: $carbon--spacing-05;
       padding-bottom: $carbon--spacing-05;
     }
     &__cards {
-      display: flex;
-      float: right;
-      flex-wrap: wrap;
-    }
-    .bx--card-link {
-      @include carbon--make-col(8, 8);
+      &__container {
+        @include carbon--make-container;
+      }
+      &__row {
+        @include carbon--make-row;
+      }
+      &__col {
+        @include carbon--make-col-ready;
+        @include carbon--make-col(4, 4);
 
-      display: flex;
-      text-decoration: none;
-      outline-offset: -1px;
+        @include carbon--breakpoint('md') {
+          @include carbon--make-col(4, 8);
+        }
+        @include carbon--breakpoint('lg') {
+          @include carbon--make-col(1, 3);
+        }
 
-      > *:not(:last-child) {
-        margin-bottom: 0;
+        margin-top: $carbon--grid-gutter--condensed / 2;
+        margin-bottom: $carbon--grid-gutter--condensed / 2;
```

Comment:
I think I saw https://github.com/carbon-design-system/carbon/pull/4960 got merged in. These lines of code may not be necessary anymore.

Reply:
you mean we could remove these props and just have `${prefix}--grid--condensed` this in JSX right?

Reply:
But we do need `margin-bottom: $carbon--grid-gutter--condensed / 2` this prop in that  I guess. Because when I remove both of them it looks like this.
![image](https://user-images.githubusercontent.com/53789187/72378722-f557cb80-36df-11ea-8f18-7e8c06fd1d02.png)

Reply:
Looks like they might not have releasee it yet for some reason. I thought they were doing nightly builds. I guess we can wait a little longer.

Reply:
Okay.